Experimental KRaft mode for development and testing (#6791)
strimzi/strimzi-kafka-operator: documentation/modules/operators/ref-operator-cluster-feature-gate-releases.adoc

* The `ControlPlaneListener` feature gate moved to beta stage in Strimzi 0.27 and is expected to remain in the beta stage until Strimzi 0.31.
* The `ServiceAccountPatching` feature gate moved to beta stage in Strimzi 0.27 and is expected to remain in the beta stage until Strimzi 0.30.
* The `UseStrimziPodSets` feature gate is currently planned to move to the beta stage in Strimzi 0.30.

> * The `UseKRaft` feature gate is currently used for development only and does not have any planned release when it will move to the beta phase.

Reviewer: ```suggestion
* The `UseKRaft` feature gate is available for development only and does not currently have a planned release for moving to the beta phase.
```
